### ISSUE TITLE
A11y_AzureToolsforVSCode_CreateNewProject_SelectaLanguage_ScreenReader: Screen reader is not announcing the place holder text in the edit field.

### ISSUE THREAD

**Issue by Swati1700** [microsoft/vscode-azurefunctions#2937]
**Actual:**
The screen reader is not announcing the place holder text "Select a Language" when focus goes on the edit field .

**Similar issue is observed in the below scenario:**
**Issue: Home** - Select a template for your projects first function
**Repro Steps:**

1. Open the application
2. Click on extension button and search for "Azure Functions"
3. Click on install and move to the "Azure" button
4. Click on "Create New Project Button" from the toolbar
5. Start the screen reader and observe the announcement by screen reader about "Select a template for your projects first function".

**Expected:**
The screen reader should announce the place holder text "Select a Language" when focus goes on the edit field .

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_AzureToolsforVSCode_Win32_Aug2021;#Azure Tools for VS Code;#Win10;#FTP;#DesktopApp;#NVDA;#JAWS;#WCAG1.3.1;#Element:EditField;

**Environment Details:** 
Application Name: Azure Tools for VS Code
Windows Version: Win10
Screen reader: NVDA2020.4, JAWS 2021

**Repro Steps:**

1. Open the application.
2. Click on extension button and search for "Azure Functions"
3. Click on install and move to the "Azure" button
4. Click on "Create New Project Button" from the toolbar
5. Start the screen reader and observe the announcement by screen reader about "Select a Language".

**User Impact:**
Screen reader users will not get correct information present on the screen.

**Comment by nturinski** [comment]
Please file as an external issue on https://github.com/microsoft/vscode/issues.  We do not control the screen reader and are using the VS Code UI components to show the place holder.

**Comment by Swati1700** [comment]
@nturinski Please find the external bug logged for above mentioned URL :
https://github.com/microsoft/vscode/issues/138114#issue-1066857016

**Comment by isidorn** [comment]
@nturinski hi there! I have closed https://github.com/Microsoft/vscode/issues/138114 because of the low quality of the  description.
Can you please verify this bug reproduces and if it does file an issue in our repository using the correct name of our UI components and try to be concise.
Thanks a lot! Without doing this it does not really scale for the vscode team to investigate into all the accessibility issues that extensions teams have.

**Comment by jinglouMSFT** [comment]
@Swati1700 I just wanted to let you know that I have tried this scenario with NVDA 2021.2, the latest release on Windows 10, "Select a language" is actually read. Here is a capture of what was read: 
![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/10081483/144148821-0f437860-1591-4779-89ea-41e2572edde7.png)

We present the same behavior as VS Code with the Git:Clone commend, which uses basically the same component. The is what was read for the Git:Clone comment (without needing any of the extensions installed to repro)
![MicrosoftTeams-image (2)](https://user-images.githubusercontent.com/10081483/144149100-ae63ce22-57cc-4c4f-bbdc-0d71af553404.png)
In this case, the text that should be read is "Provide repository URL or pick a repository source".

So the behavior between the extension and the VS Code itself is consistent.   Both watermark strings are read by NVDA 2021.2.

**Comment by Swati1700** [comment]
Verified the bug in the latest environment. Bug is fixed and does not repro. Hence closing the bug.
![image](https://user-images.githubusercontent.com/86710888/144216526-b14f5321-ea03-49a9-a1df-2d986d55b806.png)

**Comment by Rama8345** [comment]
**Github Tags:**#Closed;
